openfisca_us/variables/gov/hhs/medicaid/eligibility/medicaid_person_type.py

```diff
@@ -19,18 +21,38 @@ class medicaid_person_type(Variable):
     documentation = "Person type for Medicaid"
 
     def formula(person, period, parameters):
+        ma = parameters(period).hhs.medicaid
         # Get the person's age.
         age = person("age", period)
         # Get the existence of dependents, as defined by people 18 or younger.
         has_dependents = person.spm_unit.any(
             person.spm_unit.members("age", period) < 19
```

Comment:
could you make this a policy parameter?

Reply:
Yes, but it raises the question of how to label the FPL limit parameters and the category name

Reply:
Oh I'm not suggesting changing those, just the age limit for defining dependents.